drivers/storage/portworx/component/autopilot.go

```diff
@@ -605,6 +608,7 @@ func (c *autopilot) getAutopilotDeploymentSpec(
 	if cluster.Spec.Autopilot.Resources != nil {
 		deployment.Spec.Template.Spec.Containers[0].Resources = *cluster.Spec.Autopilot.Resources
 	}
+	deployment.Spec.Template.ObjectMeta = k8sutil.AddManagedByOperatorLabel(deployment.Spec.Template.ObjectMeta)
```

Comment:
nit: Why did we not add this in the templateLabels map directly?

Reply:
I found myself copying the nil check+initialize over and over, so I put it into this util.

Reply:
Got it 👍